pkg/arvo/app/btc-provider.hoon

```diff
@@ -75,9 +75,7 @@
     ?-  -.comm
         %set-credentials
       :_  state(host-info [api-url.comm %.n network.comm 0 *(set ship)])
-      :~  do-ping:hc
-          (start-ping-timer:hc ~s30)
-      ==
+      ~[(start-ping-timer:hc ~s30)]
```

Comment:
This is the command the provider enters to connect to the rpc server, no reason to delay 30 seconds here.

Reply:
Got it, I'll do a delay 0s. Feels good to do it through `behn` so only one code path is hitting the timers.

Reply:
Done!